specs/fault-proof/stage-one/anchor-state-registry.md

```diff
@@ -372,6 +375,20 @@ finalize withdrawals with invalidated games would be considered Critical Severit
 
 - [aASR-003](#aasr-003-incorrectly-resolving-games-will-be-invalidated-before-they-have-valid-claims)
 
+### iASR-005: The Anchor Game is recent enough to be fault provable
+
+We require that the Anchor Game corresponds to an L2 block with an L1 origin timestamp that is no
+older than 6 months from the current timestamp. This time constraint is necessary because the fault
+proof VM must walk backwards through L1 blocks to verify derivation, and processing 7 months worth
+of L1 blocks approaches the maximum time available to challengers in the dispute game process.
```

Comment:
What constrains this processing time, i.e. what would need to be improved/changed to allow older anchor games? I don't think we need to support older anchor games, just want to make sure I understand the trade off